### PR TITLE
Flush remaining output when evaluation finishes

### PR DESCRIPTION
**Pull request by ildipo**

### CODE REVIEW

File: plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java [hunk 1]
```diff
@@ -467,6 +469,29 @@ private synchronized void doAppendError(String s) {
     }
   }
 
+  private synchronized void doFlush() {
```
Comment (buob) [inline plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java:472]: Why `doFlush` instead of just `flush` or `flushOutput`?

Comment (ildipo) [inline plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java:472]: doFlush -> it does something and what it does is a flush.
this mimics other doXXX functions in the same file

Comment (buob) [inline plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java:472]: for the sake of argument, don't all functions do something? :smile:

Comment (ildipo) [inline plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java:472]: well, philosophically speaking functions compute (without side effects) and other do something with side effects - this has side effects. :-)

**Pull request merged**